src/frontend/js/components/DashBoard/utils.ts

```diff
@@ -0,0 +1,5 @@
+const getBasename = (locale: string) => {
```

Comment:
Can you just add a comment to explain the purpose of this function just to precise that it aims to be used in the dashboard context.